### DISCUSSION TITLE
向听数问题

### DISCUSSION THREAD

**Discussion by hyskylord** [Equim-chan/Mortal#10] [General]
![image](https://user-images.githubusercontent.com/26114709/180605866-ba19752f-c7a1-487d-bf47-2fcd8c4ea41e.png)
Mortal对于14张手牌的情况似乎是按照摸牌前的手牌来计算向听数的，导致如图中的情况第一次摸到听牌的时候仍然显示的是1向听。

**Comment by Equim-chan** [comment]
是的，设计上就是这样。
